packages/cli/src/services/check-parser/parser.ts

```diff
@@ -144,7 +144,8 @@ export class Parser {
 
   static readDependency (filePath: string, preferedExtenstion: SupportedFileExtension) {
     // Read the specific file if it has an extension
-    if (path.extname(filePath).length) {
+    const fileExtension = path.extname(filePath)
```

Comment:
I think we gotta add the extension to the `tryReadFileExt` instead. So if fileExtention exists, we do `Parser.tryReadFileExt(filePath, [fileExtention, ...JS_RESOLVE_ORDER])`. This is the same for TS resolve

Reply:
Mmmm, I'm not sure. Because in that case, we should explicitly add all the 'compatible extensions' to the list. In this specific case, it would be the `.page`, but maybe there is some other user using `.resource`, `.site` or any other option.
But, as you say, adding the `.page` extension into the `JS_RESOLVE_ORDER` and `TS_RESOLVE_ORDER` would fix the issue too.